CHANGELOG.md

```diff
@@ -61,6 +61,7 @@ Read more about check constraints on [Postgres Docs](https://www.postgresql.org/
 (close #3969) (#4145)
 
 ### Bug fixes and improvements
+- server: `backend_only` insert permissions (rfc #4120) (#4224)
```

Comment:
Could you take this to the highlights section? Some more info either from the RFC or the docs would help too.